src/webgl/p5.RendererGL.js

```diff
@@ -1251,16 +1282,39 @@ p5.RendererGL.prototype._getLineShader = function() {
 
 p5.RendererGL.prototype._getFontShader = function() {
   if (!this._defaultFontShader) {
-    this.GL.getExtension('OES_standard_derivatives');
+    if (this.webglVersion === constants.WEBGL) {
+      this.GL.getExtension('OES_standard_derivatives');
+    }
     this._defaultFontShader = new p5.Shader(
       this,
-      defaultShaders.fontVert,
-      defaultShaders.fontFrag
+      this._webGL2CompatibilityPrefix('vert', 'mediump') +
+        defaultShaders.fontVert,
+      this._webGL2CompatibilityPrefix('frag', 'mediump') +
```

Comment:
I am curious if this float precision is always guaranteed to work well on a device that supports WebGL2. Not sure but it is irrelevant because this design will make adjustments to precision based on the device easier in the future.